.github/workflows/integration-tests-frontend.yml

```diff
@@ -61,7 +61,74 @@ jobs:
         uses: cypress-io/github-action@v2
         with:
           working-directory: frontend
-          command: yarn run cypress:run
+          command: yarn run cypress:run -e TAGS="not @crd-validation"
+        env:
+          CYPRESS_BASE_URL: ${{ steps.set-kiali-url.outputs.kiali_url }}
+          CYPRESS_NUM_TESTS_KEPT_IN_MEMORY: 0
+          # Recorded video is unusable due to low resources in CI: https://github.com/cypress-io/cypress/issues/4722
+          CYPRESS_VIDEO: false
+
+      - name: Get debug info when integration tests fail
+        if: failure()
+        run: |
+          kubectl logs -l app.kubernetes.io/name=kiali -n istio-system
+          kubectl describe nodes
+          kubectl get pods -l app.kubernetes.io/name=kiali -n istio-system -o yaml
+
+      - name: Upload cypress screenshots when tests fail
+        uses: actions/upload-artifact@v3
+        if: failure()
+        with:
+          name: cypress-screenshots
+          path: frontend/cypress/screenshots
+  integration_tests_crd_checkers:
```

Comment:
> I probably can change the command to be yarn run cypress:run -e TAGS="not @crd-validation" && yarn run cypress:run -e TAGS="@crd-validation" to run everything in the same job. I'll change the PR to use this way.

This didn't work. When looking at the source code of that `cypress-io/github-action` I can see that [it parses the command and expects a single command](https://github.com/cypress-io/github-action/blob/f9e9419e05e5b83168d397d9c1d8d40ea179901e/index.js#L21-L52), rather than invoking it through a shell :-/

I'll add a `cypress:ci` in `package.json` as a workaround.

Reply:
> That's interesting behavior. Maybe it's just old stats sticking around after the "undo" 

I thought about that, but nope. Locally, even after killing p8s to "flush" metrics, the Passthrough cluster persisted.
Only pod restarts fix it :-(

> or else it sounds like some configs are not being updated until the pod gets restarted.

So, I lean towards this. But that would mean that Istio is not correctly updating sidecars, no? This is the reason why I think this may be uncovering some bug in Istio. I may try to identify the specific config that breaks the sidecars and report it to Istio.

Reply:
> So, I lean towards this. But that would mean that Istio is not correctly updating sidecars, no? This is the reason why I think this may be uncovering some bug in Istio. I may try to identify the specific config that breaks the sidecars and report it to Istio.

Yeah to my knowledge the sidecars should get an updated envoy config without having to restart the pods. If you can reproduce locally it's more likely a bug. The CI environment is very resource constrained so it could just be a factor of the envoy configs syncing very slowly.